### PR TITLE
ADEN-3147 Krux clean up

### PR DESCRIPTION
**Pull request by drets**
Remove wikiDirectedAtChildren and move Krux tests to php level

### CODE REVIEW

File: extensions/wikia/AnalyticsEngine/tests/AnalyticsProviderKruxTest.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+class AnalyticsProviderKruxTest extends WikiaBaseTest {
```
Comment (nandy-andy) [inline extensions/wikia/AnalyticsEngine/tests/AnalyticsProviderKruxTest.php:3]: :gem:

File: extensions/wikia/AdEngine/js/AdLogicPageParams.js [hunk 2]
```diff
@@ -210,7 +202,7 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 		}
 
 		extend(params, decodeLegacyDartParams(targeting.wikiCustomKeyValues));
-		overrideEsrbRating(params);
+		params.esrb = targeting.esrbRating || params.esrb;
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/AdLogicPageParams.js]: Don't we want to take it now always from targeting/[adContext](https://github.com/Wikia/app/blob/be0b73f3696058cacfd121efd2748d2a7239ea70/extensions/wikia/AdEngine/AdEngine2ContextService.class.php#L62)?

File: extensions/wikia/AdEngine/js/AdLogicPageParams.js [hunk 3]
```diff
@@ -210,7 +203,6 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 		}
 
 		extend(params, decodeLegacyDartParams(targeting.wikiCustomKeyValues));
-		overrideEsrbRating(params);
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/AdLogicPageParams.js:213]: `params.esrb` has to be set after `decodeLegacyDartParams` 
OR 
`decodeLegacyDartParams` should skip `esrb` occurrences

Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/AdLogicPageParams.js]: or `decodeLegacyDartParams` should skip `esrb` occurrences (to make it safer in future)

**Comment by jacekjursza** [comment]
LGTM

**Pull request merged**